cli/cobra.go

```diff
@@ -159,15 +161,31 @@ func isFirstLevelCommand(cmd *cobra.Command) bool {
 	return cmd.Parent() == cmd.Root()
 }
 
-func commandVendor(cmd *cobra.Command) string {
+func isPluginFirstLevelCommand(cmd *cobra.Command) bool {
+	return isPlugin(cmd) && isFirstLevelCommand(cmd)
+}
+
+func decoratedCommandName(cmd *cobra.Command) string {
+	decoration := " "
+	if isPluginFirstLevelCommand(cmd) {
+		decoration = "*"
+	}
+	return cmd.Name() + decoration + strings.Repeat(" ", cmd.NamePadding()-len(cmd.Name()))
+}
+
+func commandVendorAndVersion(cmd *cobra.Command) string {
 	width := 13
-	if v, ok := cmd.Annotations[pluginmanager.CommandAnnotationPluginVendor]; ok {
-		if len(v) > width-2 {
-			v = v[:width-3] + "…"
+	if vendor, ok := cmd.Annotations[pluginmanager.CommandAnnotationPluginVendor]; ok {
+		if len(vendor) > width-2 {
+			vendor = vendor[:width-3] + "…"
+		}
```

Comment:
I forgot this earlier -- now that the vendor is at the end of the line I don't think it needs to be trimmed down any more, that was just to keep the columns aligned in the old way of doing things.

Reply:
Done